apps-rendering/src/renderer.ts

```diff
@@ -196,106 +196,116 @@ const plainTextElement = (node: Node, key: number): ReactNode => {
-			textElement(format, isEditions),
-		);
-		switch (node.nodeName) {
-			case 'P':
-				return h(Paragraph, { key, format }, children);
-			case '#text':
-				return transform(text, format);
-			case 'SPAN':
-				return text;
-			case 'A':
-				return h(
-					Anchor,
-					{
-						href: withDefault('')(getHref(node)),
-						format,
-						key,
-						isEditions,
-					},
-					transform(text, format),
-				);
-			case 'H2':
-				return text.includes('* * *')
-					? h(HorizontalRule, null, null)
-					: styledH(
-							'h2',
-							{ css: HeadingTwoStyles(format), key },
-							children,
-					  );
-			case 'BLOCKQUOTE':
-				return isEditions
-					? h('blockquote', { key }, children)
-					: h(Blockquote, { key, format }, children);
-			case 'STRONG':
-				return h('strong', { key }, children);
-			case 'B':
-				return h('b', { key }, children);
-			case 'EM':
-				return h('em', { key }, children);
-			case 'BR':
-				return h('br', { key }, null);
-			case 'UL':
-				return h(List, { children });
-			case 'LI':
-				return h(ListItem, { format, children });
-			case 'MARK':
-				return styledH('mark', { key }, children);
-			case 'SUB':
-				return h('sub', { key }, children);
-			case 'SUP':
-				return h('sup', { key }, children);
-			default:
-				return null;
-		}
-	};
+const textElement = (format: Format, isEditions = false) => (
+	node: Node,
+	key: number,
+): ReactNode => {
+	const text = node.textContent ?? '';
+	const children = Array.from(node.childNodes).map(
+		textElement(format, isEditions),
+	);
+	switch (node.nodeName) {
+		case 'P':
+			return h(Paragraph, { key, format }, children);
+		case '#text':
+			return transform(text, format);
+		case 'SPAN':
+			return text;
+		case 'A':
+			return h(
+				Anchor,
+				{
+					href: withDefault('')(getHref(node)),
+					format,
+					key,
+					isEditions,
+				},
+				transform(text, format),
+			);
+		case 'H2':
+			return text.includes('* * *')
+				? h(HorizontalRule, null, null)
+				: styledH(
+						'h2',
+						{ css: HeadingTwoStyles(format), key },
+						children,
+				  );
+		case 'BLOCKQUOTE':
+			return isEditions
+				? h('blockquote', { key }, children)
+				: h(Blockquote, { key, format }, children);
+		case 'STRONG':
+			return h('strong', { key }, children);
+		case 'B':
+			return h('b', { key }, children);
+		case 'EM':
+			return h('em', { key }, children);
+		case 'BR':
+			return h('br', { key }, null);
+		case 'UL':
+			return h(List, { children });
+		case 'LI':
+			return h(ListItem, { format, children });
+		case 'MARK':
+			return styledH('mark', { key }, children);
+		case 'SUB':
+			return h('sub', { key }, children);
+		case 'SUP':
+			return h('sup', { key }, children);
+		default:
+			return null;
+	}
+};
 
 const linkColourFromFormat = (format: Format): string => {
 	if (format.theme === Special.Labs) {
 		return palette.labs[300];
 	}
 
+	if (format.design === Design.LiveBlog) {
+		return palette.neutral[100];
+	}
```

Comment:
Is this the only actual change in this file, the rest seems to be as a result of lint fix?